File: src/SIMPLEPROGRAMS/EvenNumbersUptoRange/EvenNumbersUptoRange.java
package SIMPLEPROGRAMS.EvenNumbersUptoRange;

import java.util.Scanner;

class EvenNumbersUptoRange{

    EvenNumbersUptoRange(int limt){
        int count=1;
        for(int i=0;i<=limt;i++){
            if((i%2)==0){
                System.out.println(count+") "+i);
                ++count;
            }
        }
    }
    public static void main(String[] args){

        Scanner io = new Scanner(System.in);
        System.out.print("ENTER THE LIMIT TO FIND EVEN :");

        int limt = io.nextInt();
        EvenNumbersUptoRange key = new EvenNumbersUptoRange(limt);  
    }
}

File: src/SIMPLEPROGRAMS/EvenNumbersUptoRange/EvenNumbersUptoRange.py
class even:
    def __init__(self):

        n= int(input("ENTER THE LIMT TO FIND EVEN : "))
        self.even1(n)
    def even1(self,n):
        self.n=n
        count=1
        for i in range(0,n+1,1):
            if ((i%2)==0):
                print(count,") ",i)
                count+=1

key = even()